src/VisualStudio/Core/Def/Implementation/UnusedReferences/ProjectAssets/ProjectAssetsModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
```

Comment:
I don't think this is required anymore, its the default now.

Reply:
Yup that bit is removed. This features branch was behind a bit as I developed this.